Sources/AblyAssetTrackingPublisher/Services/DefaultLocationService.swift

```diff
@@ -17,6 +17,7 @@ class DefaultLocationService: LocationService {
         }
 
         let directions = Directions(credentials: mapboxConfiguration.getCredentians())
+        NavigationSettings.shared.initialize(directions: directions, tileStoreConfiguration: .default)
```

Comment:
Why is this added here, in a PR to bump dependency version? It's a bit confusing to see this here since this line seems like it either should be present already to make it work, or it would be unnecessary.

Reply:
it's a Mapbox migration change - it needs to be here

Reply:
> it needs to be here
What happens without it?

Reply:
an exception, where Mapbox requires an entry in Info.plist